app/src/main/res/values/strings.xml

```diff
@@ -175,4 +175,10 @@
     <string name="fork_me">Fork me</string>
     <string name="cd_arrowTagImage">Arrow Tagging Game</string>
     <string name="cd_qrCode">QR Code</string>
+    <string name="invitation_title">Invite your friends to GDG app</string>
+    <string name="invitation_message">Download GDG Android App and discover Google Developers Group content while being on the go.</string>
+    <string name="invitation_deep_link" translatable="false">https://developers.google.com/groups/</string>

```

Comment:
Why did you merge it? :) I was gonna go over it one more time and addadd these event filters.